### PR TITLE
Improves the versatility of the link tag

### PR DESCRIPTION
**Pull request by mko-sci**
Fixes: OX-8823

### CODE REVIEW

**Review by idlira — CHANGES_REQUESTED** [review]
This is a breaking change, as it modifies the default behavior for links to open in a new tab. IF it's OK, the PR should be marking as breaking.
Otherwise we need the inverted behavior: one must tell that the link should open in a new tab. Lets see what others / Andi say...

File: src/main/resources/default/taglib/k/link.html.pasta [hunk 1]
```diff
@@ -1,11 +1,13 @@
+<i:arg type="String" name="id" default="" description="Defines the id."/>
 <i:arg type="String" name="link" description="Defines a url or link."/>
 <i:arg type="String" name="name" default="" description="Defines a speaking name or description."/>
 <i:arg type="String" name="icon" default="fas fa-external-link-alt" description="Defines the icon left to the link name."/>
+<i:arg type="String" name="target" default="_blank" description="Defines the target where to display the linked url or link."/>
```
Comment (jakobvogel) [inline src/main/resources/default/taglib/k/link.html.pasta:5]: `_self` seems to be a more obvious default ... or empty, and then `isFilled`

Comment (sabieber) [reply src/main/resources/default/taglib/k/link.html.pasta:5]: Would also prefer a non breaking version of this to not have to check all places where the link tag is used.

Comment (mko-sci) [reply src/main/resources/default/taglib/k/link.html.pasta:5]: okay then I am going to put "_self" as default behavior

Comment (mko-sci) [reply src/main/resources/default/taglib/k/link.html.pasta]: done

**Comment by mko-sci** [comment]
> This is a breaking change, as it modifies the default behavior for links to open in a new tab. IF it's OK, the PR should be marking as breaking. Otherwise we need the inverted behavior: one must tell that the link should open in a new tab. Lets see what others / Andi say...

Change the default behavior to _self. Therefore this change should not be breaking anymore.

**Review by idlira — APPROVED** [review]

**Review by sabieber — APPROVED** [review]

**Review by andyHa — APPROVED** [review]

**Review by jakobvogel — APPROVED** [review]

**Pull request merged**